### ISSUE TITLE
questions about the results of DynamicStereo on the SceneFlow test dataset

### ISSUE THREAD

**Issue by diligentwxx** [jiaxiZeng/Temporally-Consistent-Stereo-Matching#14]
I didn't find any results for the SceneFlow test set in the original `DynamicStereo` paper. DynamicStereo uses 20 images for inference during testing, but the SceneFlow test set only has 10 frames per video. I set `sample_len=10` and `MODEL.DynamicStereoModel.kernel_size=10`(setting it to 20 would result in an error), and the results are as follows: `EPE=0.83`, `>1px=7.99`, `>3px=3.73`, which are quite different from the results in your paper. I'd like to ask how you tested DynamicStereo?

**Comment by jiaxiZeng** [comment]
https://github.com/facebookresearch/dynamic_stereo?tab=readme-ov-file#evaluation
The author of DynamicStereo provide the ckpt trained on the sceneflow dataset, and we use it evaluate on the sceneflow cleanpass validation set. Maybe you use the final pass of the sceneflow for validation?

**Comment by diligentwxx** [comment]
Thanks for your reply. I found that I was testing the entire data of sceneflow cleanpass instead of taking 40 videos in the DynamicStereo code：`val_idxs = set(np.random.permutation(len(image_paths["left"]))[:40])`. After modification, the result are as follows: `EPE=0.74`, `>1px=7.73`, `>3px=3.51`,which is close to the result in your paper.
